pkg/gcv/cf/cf.go

```diff
@@ -168,22 +169,34 @@ func constraintAsInputData(constraintMap map[string]map[string]*configs.Constrai
 	return flattened, nil
 }
 
-func (cf *ConstraintFramework) buildRegoObject() (*rego.Rego, error) {
-	compiler, err := cf.compile()
+func (cf *ConstraintFramework) buildRegoObject(ctx context.Context) (*rego.Rego, error) {
```

Comment:
Can you briefly explain the impact of switching to the explicit rego storage interface? My concern with this approach is we're tightly coupling to the OPA internals.

Reply:
It's using the same underlying storage as it was before, this changes when it gets created by pulling it back to 'Configure' time to prevent re-creation on each "Review" call.  Simply put, this is already tightly coupled with the OPA internals, this shuffles when things happen so we can make it more efficient.

Reply:
Ok, got it.